source/common/umf_helpers.hpp

```diff
@@ -301,10 +324,9 @@ inline ur_result_t umf2urResult(umf_result_t umfResult) {
   };
 }
 
-inline umf_result_t createMemoryProvider(
+inline umf_result_t setCUMemoryProviderParams(
```

Comment:
This should be in source/adapters/cuda/common.hpp

Reply:
Moved in https://github.com/intel/llvm/pull/17015.
@igchor Done